src/Pages/Home-Page/Home.jsx

```diff
@@ -4,12 +4,24 @@ import "./responsive.css";
 import { Nav } from "../../components/Nav/Nav";
 import { NavForMobile } from "../../components/NavForMobile/NavForMobile.js";
 import { Footer } from "../../components/Footer/Footer";
+import { HomeScreen } from "./Components/HomeScreen";
+import { Category } from "./Components/Category";
+import { Offer } from "./Components/Offer";
+import { Product } from "./Components/Product";
+import { Services } from "./Components/Services";
```

Comment:
you can create a common file `index` here in components and export components from it, will be easy to access.

Reply:
Okay, I will Create Index File and Shift it.